code/modules/surgery/organs/stomach.dm

```diff
@@ -189,3 +209,17 @@
 		return
 	adjust_charge(shock_damage * siemens_coeff * 20)
 	to_chat(owner, "<span class='notice'>You absorb some of the shock into your body!</span>")
+
+/obj/item/organ/stomach/bone
+	desc = "You have no idea what this strange ball of bones does."
+
+/obj/item/organ/stomach/bone/on_life()
```

Comment:
This should probably be in the milk definition rather than checking every tick for milk